workshops/views.py

```diff
@@ -1376,18 +1376,23 @@ def workshop_issues(request):
 
     host = Role.objects.get(name='host')
     instructor = Role.objects.get(name='instructor')
-    events = Event.objects.past_events().\
-        filter(Q(attendance=None) | Q(attendance=0) |
-               Q(country=None) |
-               Q(venue=None) | Q(venue__exact='') |
-               Q(address=None) | Q(address__exact='') |
-               Q(start__gt=F('end')))
+    events = Event.objects.past_events().filter(
+        Q(attendance=None) | Q(attendance=0) |
+        Q(country=None) |
+        Q(venue=None) | Q(venue__exact='') |
+        Q(address=None) | Q(address__exact='') |
+        Q(latitude=None) | Q(longitude=None) |
+        Q(start__gt=F('end'))
```

Comment:
You can replace a number of these by using `Event.objects.past_events().unpublished_events().filter(…)` (the `venue=None` case and some others should be illegal because `Event.venue` is not nullable).  I was initially concerned that the overlap between `unpublished_events` and this filter set might be a coincidence, but the only reason we'd care about those fields for past events is probably so we can publish them (“hey, look at all the events we've held!”), so I think the overlap is meaningful and the `unpublished_events` replacement is fair.